extensions/wikia/WikiFactory/WikiFactory.php

```diff
@@ -770,7 +787,7 @@ static public function setVarById( $cv_variable_id, $city_id, $value, $reason=nu
 					 * @todo handle deleting values of this variable
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Complete the task associated to this "TODO" comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1135)